src/datasets/utils/mock_download_manager.py

```diff
@@ -126,7 +126,7 @@ def download_and_extract(self, data_url, *args):
             return self.create_dummy_data_single(dummy_file, data_url)
 
     # this function has to be in the manager under this name so that testing works
-    def download(self, data_url, *args):
+    def download(self, data_url, *args, **kwargs):
```

Comment:
I think you can remove the kwargs ?

Reply:
I left it because `*args` was already present. But I'm removing it if you think it is OK to have *args but not **kwargs.... :P